ffi/tests/eth/eth_compatibility_test.go

```diff
@@ -20,225 +20,324 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func hashData(input []byte) common.Hash {
-	return crypto.Keccak256Hash(input)
+const (
+	commit byte = iota
+	createAccount
+	updateAccount
+	deleteAccount
+	addStorage
+	updateStorage
+	deleteStorage
+	maxStep
+)
+
+var (
+	stepMap = map[byte]string{
+		commit:        "commit",
+		createAccount: "createAccount",
+		updateAccount: "updateAccount",
+		deleteAccount: "deleteAccount",
+		addStorage:    "addStorage",
+		updateStorage: "updateStorage",
+		deleteStorage: "deleteStorage",
+	}
+)
+
+type tree struct {
```

Comment:
consider renaming since we are not using a tree of proposals here and simply fuzzing a single view (committed database itself)

Reply:
Renaming to `merkleTriePair`